03Lesson/01Homework/css/style.css

```diff
@@ -1 +1,39 @@
+img {
+  width: 100%;
+  max-width: 500px;
+  float: left;
+}
 
+div.berry {
+  width: 500px;
+  height: 394px;
+  float: left;
+  background-color: #f0f0f0;
+  color: inherit;
+  display: inline;
+}
+
+.berry h1 {
+  text-align: center;
+  font-size: 1.9em;
+  color: #252b3b;
+  background-color: inherit;
+}
```

Comment:
You'll need to add one more css property to this selector to get the text vertically centered properly - what do you think that could be?